test/intent/sample1.intent.json

```diff
@@ -1,7 +1,9 @@
 {
-  "utterance": "alarm in 2 months",
-  "intent_type": "AlarmSkillCreateIntent",
-  "intent": {
-    "AlarmSkillCreateVerb": "alarm"
+  "utterance": "set an alarm for 6 pm",
+  "evaluation_timeout": 10,
```

Comment:
Is there a default evaluation timeout? Just wondering if there's a way to skip putting it on every test case.

Reply:
the default is 30 seconds. It is not needed at all and should be removed. Thanks for noticing will fix.